catalog/spatial/registry/registry-publication-action-provider/src/main/java/org/codice/ddf/registry/publication/manager/RegistryPublicationManager.java

```diff
@@ -93,10 +96,11 @@ public void init() {
           publications.put(registryId, Collections.emptyList());
         }
       }
-    } catch (PrivilegedActionException e) {
+      executorService.shutdown();
+    } catch (PrivilegedActionException | RuntimeException e) {
       LOGGER.debug(
```

Comment:
✏️ Should this be Logger.debug("Error reading from local catalog. Catalog is probably not up yet. Will try again in {} seconds.", RETRY_INTERVAL)?

Reply:
Since I am here and if people thing this is valuable information to debug, I can change it